README.md

```diff
@@ -1,126 +1,86 @@
-1. Delete the `${TF_STATE_BUCKET}`, `${VARZ_BUCKET}`, `${SEMVER_BUCKET}`, and `${BOSH_RELEASES_BLOBSTORE_BUCKET}` through the AWS console.
-
-## Validating `terraform plan` locally (beta)
-
-* Locally clone git@github.com:18F/cg-pipeline-tasks.git; you'll need `cg-pipeline-tasks/terraform-apply.sh`
-* This terraform setup won't use your AWS profile, so export your AWS secret
-  key and access key (e.g. w/
-  https://github.com/pburkholder/Dotfiles/blob/master/bin/aws_emit)
-* Set up a temp directory and get the env vars from, say, the tooling pipeline:
-```
-export TF_STACK=$(pwd)
-mkdir tmp/
-cd tmp/
-
-# set env vars from pipeline as `export key=value` for plan parameters
-eval "$(fly -t fr get-pipeline -p terraform-provision -j | jq -r '.| .jobs[] | select(.name=="plan-bootstrap-tooling") | .plan[1].params | to_entries[] | "export " + .key + "=" + .value')"
-
-# export your AWS vars, then run:
-/your/path/to/cg-pipeline-tasks/terraform-apply.sh
-
-# clean up:
-cd .. & rm -rf tmp/
-```
+# Cloud.gov Provisioning System
+
+This repository holds the terraform configuration (and BOSH vars and ops-files)
+to bootstrap our infrastructure.
+
+## Layout
+
+### Terraform
+
+This system is using Terraform 0.11.  Future versions of the main providers
+(AWS in particular) [will not work with this
+version](https://www.hashicorp.com/blog/deprecating-terraform-0-11-support-in-terraform-providers/).
+At some point, we'll need to upgrade to the (much better) 0.12 syntax.
+
+The main terraform directories are:
+
+* `modules`: where we decompose our configuration into [Terraform
+  modules](https://www.terraform.io/docs/configuration-0-11/modules.html)
+* `modules/stack/base` & `modules/stack/spoke`: the main modules that define
+  the bulk of each environment
+* `stacks`: the various "environments"
+
+#### Environments
+
+The `main` stack is a template that is used to provision the production,
+staging, and development "environments."
+
+> ##### Aside: Multiple environments in one account
+>
+> Why do I keep putting "environments" in quotes?  One confusing and suboptimal
+> point is that the various environments are isolated from each other by being
+> in separate VPCs, _not_ separate AWS accounts. This has some knock-on
+> effects.
+>
+> For one, the modules are generally built to namespace themselves:
+>
+> ``` terraform
+> resource "aws_elb" "kubernetes_elb" {
+>   name = "${var.stack_description}-kubernetes"
+>   ...
+> ```
+>
+> If we didn't do this, then the production, staging and development resources
+> would all use the same name - which wouldn't work since they're all in the same
+> AWS account.
+>
+> :shrug:
+
+The `tooling` stack contains our "proto-BOSH", which deploys the tooling BOSH.
+The tooling BOSH then deploys the BOSH directors in the main stacks.
+
+The `external` and `dns` stacks are both outside of GovCloud (commercial AWS).
+
+I believe the `test` stack is unused.
```

Comment:
It is, it would be good to put in a story to delete it and then remove this doc bit.

Reply:
I put a story in place and linked it here.